resources/dicts/thoughts/alive/apprentice.json

```
@@ -236,28 +348,221 @@
+            "charismatic",
+            "compassionate",
+            "thoughtful",
+            "confident",
+            "calm",
+            "faithful",
+            "loving",
+            "responsible",
+            "wise",
+            "cunning",
+            "sincere"
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "relationship_constraint": [
+            "app/mentor"
+        ]
+    },
+    {
+        "id": "insecure_nervous_warrior_apprentice_to_harsher_mentor",
+        "thoughts": [
+            "Seeks comfort from r_c, but doesn't get it",
+            "Feels like {PRONOUN/m_c/subject} will never live up to r_c's expectations",
+            "Hopes that training under r_c will help {PRONOUN/m_c/object} be as confident as {PRONOUN/m_c/poss} mentor",
+            "Runs off crying after getting a stern lecture by r_c",
+            "Wishes r_c could be a bit more understanding",
+            "Feels like r_c doesn't like {PRONOUN/m_c/object}",
+            "On a hard day, m_c is surprised when r_c gives {PRONOUN/m_c/object} the day off",
+            "Gets a lecture on being more confident when {PRONOUN/m_c/subject} complain about never succeeding in a move",
+            "Knows r_c is trying {PRONOUN/r_c/poss} best to help {PRONOUN/m_c/object}",
+            "Is found by r_c crying in {PRONOUN/m_c/poss} nest, and is comforted",
+            "m_c is surprised when {PRONOUN/m_c/poss} mentor brings {PRONOUN/m_c/object} {PRONOUN/m_c/poss} favorite prey",
+            "Gets lessons on posturing confidently from r_c",
+            "Gets scared of {PRONOUN/m_c/poss} mentor after r_c snaps at {PRONOUN/m_c/object}",
+            "r_c apologizes for snapping at m_c the other day"
+
+        ],
+        "main_trait_constraint": [
+            "insecure",
+            "nervous"
+        ],
+        "random_trait_constraint": [
+            "troublesome",
+            "fierce",
+            "bloodthirsty",
+            "cold",
+            "strict",
+            "ambitious",
+            "vengeful",
+            "arrogant",
+            "grumpy",
+            "rebellious"
+
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "relationship_constraint": [
+            "app/mentor"
+        ]
+    },
+    {
+        "id": "insecure_nervous_warrior_apprentice_to_outgoing_mentor",
+        "thoughts": [
+            "Wishes {PRONOUN/m_c/poss} mentor didn't encourage {PRONOUN/m_c/object} to speak with others so much",
+            "Hides behind r_c during the gathering",
+            "Wonders how r_c can speak with others so easily",
+            "Tries to do some suggestions r_c gave {PRONOUN/m_c/object} to be able to be more outgoing",
+            "Hides away from {PRONOUN/m_c/poss} mentor when {PRONOUN/m_c/subject} {VERB/m_c/learn/learns} {PRONOUN/m_c/poss} mentor wants to take {PRONOUN/m_c/object} to the far edges of the territory today"
```

seems quite long?

---

Most of that is pronoun tagging.

Hides away from their mentor when she learns their mentor wants to take them to the far edges of the territory today.

---

Would suggest cutting out one of the mentor repetitions as well? Just to help with the flow

```suggestion
            "Hides away from {PRONOUN/m_c/poss} mentor when {PRONOUN/m_c/subject} {VERB/m_c/learn/learns} {PRONOUN/r_c/subject} {VERB/r_c/want/wants} to take them to the far edges of the territory today"
```

---

Works for me :D